tycho-extras/tycho-version-bump-plugin/src/main/java/org/eclipse/tycho/versionbump/UpdateTargetMojo.java

```diff
@@ -178,4 +192,26 @@ public String getVersion() {
 
     }
 
+    private static final class Repository implements TargetDefinition.Repository {
```

Comment:
You might want to choose a more descriptive name e.g.
```suggestion
    private static final class ResolvedRepository implements TargetDefinition.Repository {
```